README.txt

```diff
@@ -39,28 +39,11 @@ following software:
 INSTALLATION:
 
 Instructions for installing Helioviewer.org are available online at
-http://wiki.helioviewer.org/wiki/Installing_Helioviewer.org
+http://wiki.helioviewer.org/wiki/index.php?title=Installing_Helioviewer.org
 
 LICENSE:
 
 Helioviewer.org is licensed under the Mozilla Public License. Software
 libraries used by Helioviewer.org and included with this distribution may
 include their own licenses. Please consult the documentation of the particular
 dependency for the details of it's licensing.
-
-CREDITS:
-
-Helioviewer.org is part of the Helioviewer Project, an open-source project
-for the visualization of solar and heliospheric data.  The Helioviewer
-Project is funded by ESA and NASA.  For more information, please see the
-wiki above.
-
-Helioviewer.org acknowledges the support of the following instrument teams,
-mission teams and institutions - SDO(NASA), SOHO(ESA/NASA), STEREO(NASA),
-PROBA2(ESA), Yohkoh(JAXA/NASA/PPARC), NASA GSFC, Royal Observatory of
-Belgium, LMSAL, SDAC, Stanford University, Montana State University/FFT
-
-Helioviewer.org acknowledges the use of the following tools:
- - www.browserstack.com, for testing the web application on multiple browsers
-   and OSs,
- - www.highcharts.com, for interactive JavaScript charts.
```

Comment:
Do we want to keep these credits? I think it's good practice/sportsmanship.

Reply:
Yes we should. Highcharts is used in the timeline functionality.